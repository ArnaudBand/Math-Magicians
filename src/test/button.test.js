import renderer from 'react-test-renderer'
import Button from '../components/button'

it('Calculator button renders correctly', () => {
  const handleClick = () => true;
  const tree = renderer.create(<Button text="Submit" handleClick={handleClick} />).toJSON();
  expect(tree).toMatchSnapshot();
})
