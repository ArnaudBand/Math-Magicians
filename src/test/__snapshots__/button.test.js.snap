// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calculator button renders correctly 1`] = `
<button
  className="class"
  id="1"
  onClick={[Function]}
  type="button"
>
  1
</button>
`;
